feat: add ability to migrate orgs (#3)
npm/pneumatic-tubes: README.md

./index.js couch-import --source-couch-db=[couch-db-url]/_changes --target-registry=[target-registry-url] --shared-fetch-secret=[password-from-console]
```

> ## Importing From npm Orgs

Reviewer: ```suggestion
## Importing From npm Orgs

### Prerequisites

You must purchase and configure an [npm Enterprise](https://www.npm-enterprise.com/) instance before you can migrate from npm Orgs to npm Enterprise.
```

> ## Importing From npm Orgs

Reviewer: What sort of permissions are necessary? Do they need to be an org admin + Enterprise admin in order to perform these steps?

Author: You need to have read access to the npm Orgs org, and write access to the npm Enterprise org.

> ## Importing From npm Orgs

Reviewer: ```suggestion
## Importing From an organization in the npm public registry
```

> ## Importing From npm Orgs

Reviewer: Suggestion for reordering steps:

1. Create an authorization token on the npm public registry
2. Create a text file containing the packages you wish to migrate
3. Log in to your npm Enterprise instance
4. Create a matching organization in npm Enterprise
5. Run the migration tool

> ### 1. Create matching organization in npm Enterprise

Reviewer: ```suggestion
### 1. Create a matching organization in npm Enterprise
```

You should create an organization in npm Enterprise that matches the name of

> the organization that you wish to migrate from in npm Orgs.

Reviewer: ```suggestion
the organization that you wish to migrate from in the public registry.
```

> As an example, if you want to migrate from your organization `@babel` on the

Reviewer: Should both references to babel be `@babel` here?

public registry, create an organization named `babel` in npm Enterprise.

> ### 2. Create an authorization token

Reviewer: ```suggestion
### 2. Create an authorization token on the npm public registry
```

To migrate from npm Orgs, you need an authorization token from the public
registry.

> 1. visit `https://www.npmjs.com`.

Reviewer: ```suggestion
1. Visit https://www.npmjs.com.
```

> 2. in the account drop down in the upper right, choose the option "Tokens".

Reviewer: ```suggestion
2. In the account drop down in the upper right corner of the page, click "Tokens".
```

> 3. click "Create New Token", and copy this somewhere safe for later.

Reviewer: ```suggestion
3. Click "Create New Token", then copy the generated token to a local text file.
```

> ### 3. Login to your npm Enterprise Instance

Reviewer: ```suggestion
### 3. Login to your npm Enterprise instance
```

> 

Reviewer: ```suggestion
As an npm Enterprise administrator, log in to your instance:
```

```
npm config set registry https://registry.my-instance.npme.io
npm login
```

> ### 4. Create a text file containing the packages you wish to migrate

Reviewer: ```suggestion
### 4. Create a text file containing the packages in the organization you wish to migrate
```

> _Note: you can only publish scoped modules to your private registry._

Reviewer: ```suggestion
_Note: you can only publish scoped packages to your private registry._
```

> The text file should look something like this:

Reviewer: Can this have commas or other line separators, or does each entry need a new line?

Author: Each entry should be a new line.

```
@babel/runtime
@babel/core
@babel/template
```

### 5. Run the migration tool

It's now time to run the migration tool, using the token and text file that
you generated.

> _Note: this will migrate all versions of your module._

Reviewer: ```suggestion
_Note: this will migrate all versions of each package listed in the text file._
```
